airbyte-integrations/connectors/destination-elasticsearch/src/main/java/io/airbyte/integrations/destination/elasticsearch/ElasticsearchConnection.java

```diff
@@ -56,7 +57,17 @@ public ElasticsearchConnection(ConnectorConfiguration config) {
 
     // Create the low-level client
     httpHost = HttpHost.create(config.getEndpoint());
-    restClient = RestClient.builder(httpHost)
+    final RestClientBuilder builder = RestClient.builder(httpHost);
+
+    // Set custom user's certificate if provided
+    if (config.getCertAsBytes() != null && config.getCertAsBytes().length != 0){
```

Comment:
Please take a look at how similar functionality is implemented for JDBC connectors (Postgres source, destination, and MySQL source). Let's keep these implementations as consistent as possible. One question that is not clear to me here is whether the format of the certificate that user has to paste into UI for Elastic Search is different from Postgres and MySQL and if it is - can this be avoided?

Reply:
Hi @grishick.
I will check about the format, but could you please clarify the first part of your comment? Other DBs are based on JDBC client meanwhile in Elasticsearch we use a native SDK RestClient builder.
Thanks

Reply:
Updated to use the same format. Thanks